tests/unit/amo/pages/TestUsersUnsubscribe.js

```diff
@@ -141,9 +124,11 @@ describe(__filename, () => {
     });
     errorHandler.handle(new Error('unexpected error'));
```

Comment:
We have a new pattern for this, which uses `createFailedErrorHandler`. Look for one that also uses the `message` argument to provide a specific message to look for.